src/store.js

```diff
@@ -0,0 +1,81 @@
+export const TOGGLE = 'action/toggle';
+export const LOAD_TODOS = 'action/load_todos';
+export const SWAP_TODOS = 'action/swap';
+export const ADD_TODO = 'action/add_todo';
+export const EDIT_TODO = 'action/edit_description';
+export const DELETE_TODO = 'action/delete_todo';
+export const CLEAR_COMPLETED = 'action/clear_completed';
```

Comment:
You can try exporting the whole class with static methods and avoid multiple exports. So you can easily access all your functionality and save some code lines.